### PR TITLE
 Adding doc template to funtion polyval3

### PR DESCRIPTION
**Pull request by Daniel-Tojal**
An input and an output are to be determined

### CODE REVIEW

File: src/Functions/polyval3.m [hunk 1]
```diff
@@ -1,11 +1,21 @@
...
 function val = polyval3(polyCoeffs, evalAt)
+% evaluate an (independent component) polynomial in R3
+%
+% USAGE:
+%
+%    val = polyval3(polyCoeffs, evalAt)
+%
+% INPUTS: 
+%    polyCoeffs:    Coefficient matrix
+%    evalAt:
```
Comment (Daniel-Tojal) [inline src/Functions/polyval3.m]: @adhusch Could you please provide a description of the input and the output?

Comment (adhusch) [reply src/Functions/polyval3.m]: This method evaluates a function (precisely a polynomial mapping from R^1 -> R^3) i.e. sort of y=f(x) where x \in R^1 and y \in R^3 at the given point x="evalAT".

**Comment by laurentheirendt** [comment]
@Daniel-Tojal

**Comment by laurentheirendt** [comment]
Thanks @Daniel-Tojal 👍

**Pull request merged**